### PR TITLE
[Robot] adding review donations and update opportunity testcase for GE

### PR DESCRIPTION
**Pull request by skristem**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/resources/NPSP.py [hunk 1]
```diff
@@ -248,31 +248,31 @@ def select_related_row(self, value):
...
         locator=npsp_lex_locators['click_aff_id'].format(self.aff_id_text)
-        self.selenium.get_webelement(locator).click()   
-        
-#     @capture_screenshot_on_error    
+        self.selenium.get_webelement(locator).click()
+
+#     @capture_screenshot_on_error
 #     def navigate_to_and_validate_field_value(self, field,status,value,section=None):
 #         """If status is 'contains' then the specified value should be present in the field
 #                         'does not contain' then the specified value should not be present in the field
```
Comment (Safiyab) [inline robot/Cumulus/resources/NPSP.py:278]: Is there a specific reason you commented this code but not delete it ?

File: robot/Cumulus/resources/GiftEntryPageObject.py [hunk 2]
```diff
@@ -341,3 +341,18 @@ def verify_table_field_values(self,table,**kwargs):
             locator=npsp_lex_locators["gift_entry"]["table"].format(table,field,value)
             self.selenium.wait_until_page_contains_element(locator,error=f'{field} does not contain {value} in {table} table')
 
+    def verify_link_status(self, **kwargs):
+        """ Verify the link is disabled/enabled, pass the name of the link
+        and the expected status of the link as either enabled or disabled"""
```
Comment (rjanjanam) [inline robot/Cumulus/resources/GiftEntryPageObject.py:346]: Please provide an example of the usage to support the keyword documentation

File: robot/Cumulus/resources/GiftEntryPageObject.py [hunk 3]
```diff
@@ -341,3 +341,18 @@ def verify_table_field_values(self,table,**kwargs):
...
+        for key,value in kwargs.items():
+            locator = npsp_lex_locators["button-with-text"].format(key)
+            self.selenium.wait_until_element_is_visible(locator,error= f"'{key}' is not displayed on the page")
+            if value=="disabled":
+                actual_value = self.selenium.get_webelement(locator).get_attribute(value)
+                if actual_value == None or actual_value == False:
```
[the remark this answers is not in the document]
Comment (skristem) [reply robot/Cumulus/resources/GiftEntryPageObject.py:352]: no in this case there is an attribute called disabled that is displayed on the element if its disabled. If its enabled u don't see the disabled attribute.

File: robot/Cumulus/resources/NPSP.py (hunk 1 above)
Comment (skristem) [reply robot/Cumulus/resources/NPSP.py:278]: this was old code I made a copy and experimented with it but forgot to delete at that time, I will delete this

**Pull request merged**